CephHealthException error in test_add_capacity_node_restart

The below test case is failing with CephHealthException while waiting for ceph health status to be HEALTH_OK.
```
tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py::TestAddCapacityNodeRestart::test_add_capacity_node_restart[1-workload_storageutilization_rbd0]
```
Result - https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/16690/testReport/

Error details:
```
# Verify OSDs are encrypted
        if config.ENV_DATA.get("encryption_at_rest"):
            osd_encryption_verification()
    
        logging.info("Finished verifying add capacity osd storage with node restart")
        logging.info("Waiting for ceph health check to finished...")
>       ceph_health_check(namespace=config.ENV_DATA["cluster_namespace"], tries=90)

tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py:90: 
```
```
if health.strip() == "HEALTH_OK":
            log.info("Ceph cluster health is HEALTH_OK.")
            return True
        else:
>           raise CephHealthException(f"Ceph cluster health is not OK. Health: {health}")
E           ocs_ci.ocs.exceptions.CephHealthException: Ceph cluster health is not OK. Health: HEALTH_WARN Degraded data redundancy: 31756/145263 objects degraded (21.861%), 55 pgs degraded, 49 pgs undersized

ocs_ci/utility/utils.py:1621: CephHealthException
```
Add capacity succeeded. But recovery is taking more than the given wait time.

--- a/tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py
+++ b/tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py
@@ -87,4 +87,4 @@ class TestAddCapacityNodeRestart(ManageTest):
 
         logging.info("Finished verifying add capacity osd storage with node restart")
         logging.info("Waiting for ceph health check to finished...")
-        ceph_health_check(namespace=config.ENV_DATA["cluster_namespace"], tries=90)
+        ceph_health_check(namespace=config.ENV_DATA["cluster_namespace"], tries=180)

